src/main/java/com/tupilabs/testng/parser/TestNGParser.java

```diff
@@ -50,16 +51,16 @@
 	 * Parses the content of an input stream and returns a Suite.
 	 * 
 	 * @param inputStream the input stream.
-	 * @return Resulting object.
+	 * @return List of Resulting object.
 	 * @throws SAXException
 	 * @throws IOException
 	 * @throws ParserConfigurationException
 	 */
-	public Suite parse(File file) throws ParserException {
+	public List<Suite> parse(File file) throws ParserException {
 
 		FileInputStream fileInputStream = null;
 		final TestNGXmlHandler handler = new TestNGXmlHandler();
-		final Suite suite;
+		//final Suite suite;
```

Comment:
Let's remove this line. git will keep track of old code like this (though I do leave some code commented when it's something really complicated)